sdcm/sct_events/base.py

```diff
@@ -497,6 +498,8 @@ def msgfmt(self):
             fmt += " line_number={0.line_number}"
         if self.node is not None:
             fmt += " node={0.node}"
+        if hasattr(self, "known_issue") and self.known_issue:
```

Comment:
I won't if we could make it a bit more visible...

since people won't notice it, if they are not looking for it....